spec/services/catalog/create_request_body_from_spec.rb

```diff
@@ -25,12 +34,14 @@
           :product   => order_item.portfolio_item.name,
           :portfolio => order_item.portfolio_item.portfolio.name,
           :order_id  => order_item.order_id.to_s,
+          :platform  => Catalog::CreateRequestBodyFrom.new(order_item.order, order_item, 1).send(:platform, order_item.portfolio_item),
```

Comment:
Should be able to change this to `'the platform'` now, right? Stops us from attempting to call a private method on a class that's already under test.